Documentation/ApiOverview/ContentSecurityPolicy/Index.rst

```diff
@@ -104,8 +294,35 @@ example:
 ..  literalinclude:: _ContentSecurityPolicies.php
     :caption: EXT:my_extension/Configuration/ContentSecurityPolicies.php
 
-..  todo: Explain "Scope", "MutationCollection", "Mutation", "MutationMode", ...
+The API here is much like the YAML syntax. The PHP code needs to return
+an mapped array of an :php:`MutationCollection` instance with all rules
+put into a sub-array, containing instances of a single :php:`Mutation`.
 
+Each :php:`Mutation` instance is like a Data Object (DO) where it's constructor
+allows you to specifiy a `mode` (type :php:`MutationMode`), a `directive`
+(type :php:`Directive`) and one ore more actual values ("sources", type :php:`UriValue`
+or `SourceKeyword`).
```

Comment:
Should we add the FQCN to all the classes/enums mentioned?

Reply:
Yes. Sadly this is beyond the scope of what I can offer to implement right now, so I only added a TODO for this. Hopefully we can get to it when we move the sections!

Reply:
Fine for me.